Tetris/main.qml

```diff
@@ -0,0 +1,78 @@
+import QtQuick 2.12
+import QtQuick.Window 2.12
+import Qt.labs.settings 1.0
+
+Window {
+    id: mainWin
+    visible: true
+    width: 700
+    height: 550
+    title: qsTr("Tetris")
+    maximumHeight: height
+    minimumHeight: height
+    maximumWidth: width
+    minimumWidth: width
+    Settings {
+           property alias x: mainWin.x
+           property alias y: mainWin.y
+       }
+
+    Init{}
```

Comment:
please find a more understandable name for this item, Init sound more like a function but it contains items